src/main/c/sslcontext.c

```diff
@@ -1340,16 +1340,6 @@ TCN_IMPLEMENT_CALL(void, SSLContext, setSessionTicketKeys)(TCN_STDARGS, jlong ct
     int i;
     int cnt;
 
-    if (((*e)->GetArrayLength(e, keys) % SSL_SESSION_TICKET_KEY_SIZE) != 0) {
```

Comment:
I think it is completely fine to only check in the caller as it is only done by our internal stuff. We do the same kind of thing in other native methods in netty.